CC3D/Models/BiocIU/SARSCoV2MultiscaleVTM/Model/Models/readme.rst

```diff
@@ -59,21 +59,72 @@ framework. This can be accomplished as follows, using the previous example of de
+CC3D.
+``nCoVSteppableBase`` provides shared data with all steppables for making their unit conversions, and so that
+you can specify data once that all steppables respond to for consistent specification of units among all steppables
+(rather than explicitly informing each steppable that you use in your simulation).
+
+Currently, ``nCoVSteppableBase`` provides the following attributes that you can get or set on any instances of a class
+that uses ``nCoVSteppableBase``, the changes to which will be reflected in all steppables that use
+``nCoVSteppableBase``,
+
+- ``step_period``: the period of one simulation step, in units of seconds per step (default 60).
+- ``voxel_length``: the length of the side of each voxel, in units microns per voxel side length (default 1).
+
+You can see an example of this in the default simulation specification distributed with the framework. The default
+configuration specifies that the simulation consists of voxels of size 4 x 4 x 4 microns and five minutes per
+simulation step. These operations can be found in ``Simulation/ViralInfectionVTM.py``,
+
+.. code-block:: python
+
+    from ViralInfectionVTMSteppables import CellInitializerSteppable
+    steppable = CellInitializerSteppable(frequency=1)
+    steppable.voxel_length = 4.0
+    steppable.step_period = 5.0 * 60
+    CompuCellSetup.register_steppable(steppable=steppable)
+
+To be completely clear, you could perform these operations on any steppable that uses ``nCoVSteppableBase``, like
+``CellInitializerSteppable`` in this example, and the changes will be reflected in all steppables so long as you make
+them `before` calling ``CompuCelLSetup.run()``.
+We will make every effort to ensure that all steppables in the shared library respond appropriately to such operations,
+so that you don't need to worry about whether steppables are performing their model specification with the unit
+specification that you prescribe.
+We also recommend that you use these attributes in your own steppables that use ``nCoVSteppableBase``, and especially
+should you decide to contribute your own model steppables to the shared library.
+
 Framework Event System
 ----------------------
 
-``nCoVSteppableBase`` is a special class that allows you to exploit both the basic functionality of cc3d `and` the
-functionality of the shared models that are distributed with this framework.
-In particular, say you've deployed a steppable from the shared library that creates a new cell, and say that you'd
+Say you've deployed a steppable from the shared library that creates a new cell, and say that you'd
 like to do something with that cell when it is created. How would you know when it was created?
 The computationally expensive solution is to check the cell inventory and look for changes.
 However, some steppables in the shared library also issue procedures during such events, and tracking down where all
```

Comment:
> and tracking down where _all each_ steppable does such things can be tedious, confusing and even more computationally expensive.

typo

Reply:
Corrected in https://github.com/covid-tissue-models/covid-tissue-response-models/pull/80/commits/a17b2f2813ff1131091753f82801d90c801a1cc2